vulnfeeds/cves/nvd2.go

```diff
@@ -0,0 +1,399 @@
+// Code generated by github.com/atombender/go-jsonschema, DO NOT EDIT.
+
+package cves
+
+import "encoding/json"
+import "fmt"
+import "strings"
+import "time"
+
+import "github.com/google/osv/vulnfeeds/cvss20"
+import "github.com/google/osv/vulnfeeds/cvss30"
+import "github.com/google/osv/vulnfeeds/cvss31"
+import "github.com/atombender/go-jsonschema/pkg/types"
+
+type Config struct {
+	Operator string `json:"operator,omitempty" yaml:"operator,omitempty" mapstructure:"operator,omitempty"`
+	Negate bool `json:"negate,omitempty" yaml:"negate,omitempty" mapstructure:"negate,omitempty"`
+	Nodes []Node2 `json:"nodes" yaml:"nodes" mapstructure:"nodes"`
+}
+
+// CPE match string or range
+type CpeMatch struct {
```

Comment:
nit: these acronyms should all be uppercase per https://google.github.io/styleguide/go/decisions#initialisms

Reply:
Addressed in 4be1eeb